beacon/goclient/attest.go

```diff
@@ -324,68 +345,170 @@ func (gc *GoClient) fetchWeightedAttestationData(ctx context.Context,
 // scoreAttestationData generates a score for attestation data.
 // The score is relative to the reward expected from the contents of the attestation.
 func (gc *GoClient) scoreAttestationData(ctx context.Context,
+	client Client,
 	attestationData *phase0.AttestationData,
 	logger *zap.Logger,
 ) float64 {
-	logger = logger.With(
-		fields.BlockRoot(attestationData.BeaconBlockRoot),
-		zap.Uint64("attestation_data_slot", uint64(attestationData.Slot)))
 	// Initial score is based on height of source and target epochs.
 	score := float64(attestationData.Source.Epoch + attestationData.Target.Epoch)
+	logger.
+		With(zap.Float64("base_score", score)).
+		Info("base score was set. Fetching slot for block root")
```

Comment:
This `.Info()` log for base score might be a bit noisy since it runs per client per attestation
maybe drop it to `.Debug()`?